android/src/main/java/com/rudderstack/react/android/RNRudderSdkModule.java

```diff
@@ -147,6 +147,32 @@ public void identify(String userId, ReadableMap traits, ReadableMap options) {
         rudderClient.identify(userId, Utility.convertReadableMapToTraits(traits), Utility.convertReadableMapToOptions(options));
     }
 
+    @ReactMethod
+    public void alias(String newId, ReadableMap options) {
+        if (rudderClient == null) {
+            RudderLogger.logWarn("Dropping the Alias call as RudderClient is not initialized yet, Please use `await` keyword with the setup call");
+            return;
+        }
+        if (TextUtils.isEmpty(newId)) {
+            rudderClient.alias(Utility.convertReadableMapToOptions(options));
+            return;
+        }
+        rudderClient.alias(newId, Utility.convertReadableMapToOptions(options));
+    }
+
+    @ReactMethod
+    public void group(String groupId, ReadableMap traits, ReadableMap options) {
+        if (rudderClient == null) {
+            RudderLogger.logWarn("Dropping the Group call as RudderClient is not initialized yet, Please use `await` keyword with the setup call");
+            return;
+        }
+        if (TextUtils.isEmpty(groupId)) {
+            rudderClient.group(Utility.convertReadableMapToTraits(traits), Utility.convertReadableMapToOptions(options));
```

Comment:
We are not having group(traits, options) API as well in Android, can you please change it to use a valid Group API?